.env

```diff
@@ -0,0 +1 @@
+VITE_SANITY_PROJECT_ID=6yf8mcap
```

Comment:
Pamietaj ze nigdy nie dajesz tajnych danych z .env do githuba. W ogole nie powinno tutaj byc tego pliku. Kazdy lokalnie sam powinien go stworzyć. Mozna za to zrobic .env.example i tam wpisac takie wartosci jak VITE_SANITY_PROJECT_ID="" zeby po prostu inni wiedzieli co musza wypełnić. Pozniej na podstawie tego pliku (env.example) kazdy wygeneruje swoj własny plik env